### PR TITLE
[China Network] Update DNS options

### PR DESCRIPTION
**Pull request by bjin-cloudflare**
### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

**Comment by bjin-cloudflare** [comment]
@pedrosousa - would you please review this pull request for China Network?

File: src/content/docs/china-network/concepts/china-dns.mdx [hunk 1]
```diff
@@ -6,20 +6,54 @@ sidebar:
 
 ---
 
+# Default
+
+By default Cloudflare China Network provides a nearest-to-client nameservice. The DNS request is resolved on the data center nearest to the client out of China. And the Http request is served on the data center nearest to the client. Cloudflare data center out of China is adopted for clients out of China, and JD Cloud data center in China for clients in China.
```
Comment (crwaters16) [inline src/content/docs/china-network/concepts/china-dns.mdx]:
```suggestion
Cloudflare China Network provides a nearest-to-client nameservice by default. The DNS request is resolved on the data center nearest to the client out of China, and the HTTP request is served on the data center nearest to the client. The closest Cloudflare data center outside of China is adopted for clients outside of China, and the JD Cloud data center in China is adopted for clients in China.
```

File: src/content/docs/china-network/concepts/china-dns.mdx [hunk 2]
```diff
@@ -6,20 +6,54 @@ sidebar:
...
+|                     |               |
+
+
+
 ## General setup
 
 After you [enable the Cloudflare China Network service](/china-network/get-started/), do the following:
 
-1. Contact your account team to enable the feature. Currently, you cannot enable China Authoritative DNS in the Cloudflare dashboard.
+1. You will need to reach out to your Cloudflare sales team to enable the feature. Currently the feature can't be enabled from the Cloudflare dashboard.
```
Comment (crwaters16) [inline src/content/docs/china-network/concepts/china-dns.mdx]:
```suggestion
1. Contact your account team to enable the feature. Currently, you cannot enable China Authoritative DNS in the Cloudflare dashboard.
```

^this is standard language we use throughout documentation when we ask users to contact us, just fyi!

File: src/content/docs/china-network/concepts/china-dns.mdx [hunk 3]
```diff
@@ -6,20 +6,54 @@ sidebar:
...
 
 Before you enable China Authoritative DNS, you should confirm that majority (over 90%) of the traffic is coming from China. After you enable China Authoritative DNS, all the global DNS requests will be routed to JD Cloud data centers in China instead of the nearest data centers.
 
+## Comparison
+
+Default vs. In-China Nameserver
+
+| **DNS Option**          | **Behavior** |
+|---------------------|---------------|
+| Default             | Making use of the nearest DNS server available to eyeball. |
```
Comment (crwaters16) [inline src/content/docs/china-network/concepts/china-dns.mdx]:
```suggestion
| Default             | Uses the DNS server closest to the end user. |
```

File: src/content/docs/china-network/concepts/china-dns.mdx [hunk 4]
```diff
@@ -6,20 +6,54 @@ sidebar:
...
+| In-China Nameserver | Only use DNS in Mainland China operated by JD Cloud. |
```
Comment (crwaters16) [inline src/content/docs/china-network/concepts/china-dns.mdx]:
```suggestion
| In-China Nameserver | Uses only DNS in China, operated by JD Cloud. |
```

File: src/content/docs/china-network/concepts/china-dns.mdx [hunk 5]
```diff
@@ -6,20 +6,54 @@ sidebar:
 
 ---
 
+# Default
```
Comment (crwaters16) [inline src/content/docs/china-network/concepts/china-dns.mdx]:
```suggestion
```

File: src/content/docs/china-network/concepts/china-dns.mdx [hunk 6]
```diff
@@ -6,20 +6,54 @@ sidebar:
...
+
+By default Cloudflare China Network provides a nearest-to-client nameservice. The DNS request is resolved on the data center nearest to the client out of China. And the Http request is served on the data center nearest to the client. Cloudflare data center out of China is adopted for clients out of China, and JD Cloud data center in China for clients in China.
+
+
+# In-China Nameserver
```
Comment (crwaters16) [inline src/content/docs/china-network/concepts/china-dns.mdx]:
```suggestion
## In-China Nameserver
```

File: src/content/docs/china-network/concepts/china-dns.mdx [hunk 7]
```diff
@@ -6,20 +6,54 @@ sidebar:
...
+
+
 ## General setup
 
 After you [enable the Cloudflare China Network service](/china-network/get-started/), do the following:
 
-1. Contact your account team to enable the feature. Currently, you cannot enable China Authoritative DNS in the Cloudflare dashboard.
+1. You will need to reach out to your Cloudflare sales team to enable the feature. Currently the feature can't be enabled from the Cloudflare dashboard.
+
+    The current China Network supports both a [full setup](/dns/zone-setups/full-setup/) and a [partial setup](/dns/zone-setups/partial-setup/).
```
Comment (crwaters16) [inline src/content/docs/china-network/concepts/china-dns.mdx]:
```suggestion
    The Cloudflare China Network currently supports domains using a [full setup](/dns/zone-setups/full-setup/) or a [partial setup](/dns/zone-setups/partial-setup/).
```

File: src/content/docs/china-network/concepts/china-dns.mdx [hunk 8]
```diff
@@ -6,20 +6,54 @@ sidebar:
...
+
+            CNAME `media.samplezone.cn` to `media.samplezone.cn.cdn.cloudflarecn.net`
 
-   The Cloudflare China Network currently supports domains using a [full setup](/dns/zone-setups/full-setup/) or a [partial setup](/dns/zone-setups/partial-setup/).
-2. Update your domain registrar with the assigned in-China nameservers. These nameservers are displayed in the Cloudflare dashboard.
 3. Test your configuration by checking if the domain resolves correctly.
 
-For further assistance, contact your account team.
+
+For further assistance, contact your sales team.
```
Comment (crwaters16) [inline src/content/docs/china-network/concepts/china-dns.mdx]:
```suggestion
For further assistance, contact your account team.
```

File: src/content/docs/china-network/index.mdx [hunk 9]
```diff
@@ -20,6 +20,7 @@ The Cloudflare China Network provides:
 * A single solution for both performance improvement and security services such as [WAF](/waf/), [DDoS](/ddos-protection/), and [bot management](/bots/).
 * An unified experience for managing network traffic and security posture. You can manage all configurations on the same dashboard.
 * The same customer support capabilities as Cloudflare’s global network. You may also have access to premium service and local language support.
+* [In-China Authoritative DNS and in-China nameservers](/china-network/concepts/china-dns/) improve the Time to First Byte (TTFB) performance.
```
Comment (crwaters16) [inline src/content/docs/china-network/index.mdx]:
```suggestion
* [In-China Authoritative DNS and in-China nameservers](/china-network/concepts/china-dns/) to improve the Time to First Byte (TTFB) performance.
```

File: src/content/docs/china-network/concepts/china-dns.mdx [hunk 10]
```diff
@@ -6,20 +6,54 @@ sidebar:
...
 
 After you [enable the Cloudflare China Network service](/china-network/get-started/), do the following:
 
-1. Contact your account team to enable the feature. Currently, you cannot enable China Authoritative DNS in the Cloudflare dashboard.
+1. You will need to reach out to your Cloudflare sales team to enable the feature. Currently the feature can't be enabled from the Cloudflare dashboard.
+
+    The current China Network supports both a [full setup](/dns/zone-setups/full-setup/) and a [partial setup](/dns/zone-setups/partial-setup/).
+2. Update your domain registrar with the assigned in-China nameservers.
+
+    a. Full setup: These nameservers are displayed in the Cloudflare dashboard.
```
Comment (crwaters16) [inline src/content/docs/china-network/concepts/china-dns.mdx]:
```suggestion
   * Full setup: These nameservers are displayed in the Cloudflare dashboard.
```

File: src/content/docs/china-network/concepts/china-dns.mdx [hunk 11]
```diff
@@ -6,20 +6,54 @@ sidebar:
...
+
+    b. Partial setup: CNAME to \<host_name\>.cdn.cloudflarecn.net.
```
Comment (crwaters16) [inline src/content/docs/china-network/concepts/china-dns.mdx]:
```suggestion
    * Partial setup: CNAME to \<host_name\>.cdn.cloudflarecn.net.
```

File: src/content/docs/china-network/concepts/china-dns.mdx [hunk 12]
```diff
@@ -6,20 +6,54 @@ sidebar:
...
+
+    For example:
+            You have a zone named samplezone.cn. In this zone, you have 2 DNS records: www and media pointing to 2 different Origins. Your Authoritative DNS server needs to set
```
Comment (crwaters16) [inline src/content/docs/china-network/concepts/china-dns.mdx]:
```suggestion
            You have a zone named samplezone.cn. In this zone, you have two DNS records: www and media, pointing to two different origins. Your Authoritative DNS server needs to set:
```

**Review by crwaters16 — CHANGES_REQUESTED** [review]
Hey Bill - requested a few changes to align with our style guide. Will approve once they're addressed!

**Comment by bjin-cloudflare** [comment]
> Hey Bill - requested a few changes to align with our style guide. Will approve once they're addressed!

Thank you Claire! All suggestions are accepted.

**Review by crwaters16 — APPROVED** [review]

**Pull request merged**